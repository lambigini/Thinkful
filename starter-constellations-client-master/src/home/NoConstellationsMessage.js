import React from "react";

function NoConstellationsMessage () {
return (
  <div className="container my-5">
    <div className="row">
      <div className="col">
        <p className="text-center">There are no constellations listed...</p>
      </div>
    </div>
  </div>
);
}

export default NoConstellationsMessage;
