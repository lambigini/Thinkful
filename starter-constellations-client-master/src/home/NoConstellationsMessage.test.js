import React from "react";
import NoConstellationsMessage from "./NoConstellationsMessage";
import renderer from "react-test-renderer";

it("renders the NoConstellationsMessage", () => {
  const component = renderer.create(<NoConstellationsMessage />);
  expect(component.toJSON()).toMatchSnapshot();
});
