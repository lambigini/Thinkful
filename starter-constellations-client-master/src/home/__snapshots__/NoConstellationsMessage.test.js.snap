// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NoConstellationsMessage 1`] = `
<div
  className="container my-5"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <p
        className="text-center"
      >
        There are no constellations listed...
      </p>
    </div>
  </div>
</div>
`;
